bin/lerna.js

```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env node
 
 var args = process.argv.slice(2);
-if (args.length > 1) {
```

Comment:
Hey thanks for the comment. How would you like to see this part? Perhaps the check could be moved to the individual command scripts?